src/styles/experimental/_grid.scss

```diff
@@ -29,6 +29,23 @@
     & > * {
       margin: $default-margin / 2;
       width: 45%;
+
+      &.dg_icon-link {
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        text-align: left;
+        padding: $padding-small;
+      }
+
+      &.dg_icon-link.dg_icon-link--large i {
+        padding: $padding-small $default-padding $padding-small $padding-small;
+        line-height: 1;
+        height: auto;
+        min-width: 75px;
+        text-align: center;
+        width: auto;
+      }
     }
```

Comment:
Does this add `font-size: 18px;` to `.dg_icon-link` and `font-size: 2em;` to` .dg_icon-link.dg_icon-link--large i` as well, or was that updated in another pr?

Reply:
@theilman7 i copied this over, and this needs updated.